assets/src/edit-story/app/media/utils/useTranscodeVideo.js

```diff
@@ -28,8 +28,37 @@ import { useCurrentUser } from '../../currentUser';
 
 const isDevelopment = process.env.NODE_ENV === 'development';
 
+/**
+ * Returns file basename without extension.
+ *
+ * @param {File} file File object.
+ * @param {string} file.name File name.
+ * @return {string} File name without extension.
+ */
 const getFileName = ({ name }) => name.split('.').slice(0, -1).join('.');
```

Comment:
Could we use [getFileName](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/utils/getFileName.js) here. We could just call it like this 

```
const outputFileName = getFileName(file.name) + '.mp4';
```

Reply:
As far as I can tell, they're not the same.

That util you linked to returns the file name from a given path, while this function here returns the file basename without the extension.

Other function: `/foo/bar.txt` -> `bar.txt`

This function: `foo.mp4` -> `foo`

It doesn't help that the other function isn't documented and not tested at all.

Reply:
I remembered we had that util, wonder if we can save a couple of bytes here. But seems like we can't :( 

I think the place I was remember getting the file name with ext was [here](https://github.com/google/web-stories-wp/blob/bf9853a66cfb7da4857304391aa29fa062406e0a/tests/e2e/utils/uploadMedia.js#L40). It is a little cleaner, if you want to use it, then go for it.